src/clients/starknet-tx/index.ts

```diff
@@ -1,34 +1,46 @@
-import { AddTransactionResponse, Contract, defaultProvider as provider, hash } from 'starknet';
+import { Account, ec, AddTransactionResponse, Contract, defaultProvider as provider, hash } from 'starknet';
 import abi from './abi/auth.json';
 import constants from './constants.json';
-import { strToShortStringArr } from '../../utils/strings';
+import { strToShortStringArr, shortStrToFelt } from '../../utils/strings';
 
 const { getSelectorFromName } = hash;
 
 export class StarkNetTx {
+
+  public auth: Contract;
+
+  constructor() {
+    // @ts-ignore
+    this.auth = new Contract(abi as Abi, constants.auth, provider);
+    const starkKeyPair = ec.getKeyPair(process.env.PK as string)
+    const account = new Account(provider, process.env.ADDRESS as string, starkKeyPair)
```

Comment:
We shouldn't have process.env in the code, this lib will be used on frontend too. Instead we should pass the account in the constructor.